src/content/docs/usage/workspaces.mdx

```diff
@@ -105,11 +105,21 @@ Out of the box, Daytona can be configured to open your Workspace in the followin
     *   WebStorm
 *   Terminal SSH
 
+### Set the Default IDE
+You can set the default IDE used by Daytona.
+After setting the default IDE, it will be opened automatically and connected to your Workspace when using the `daytona open` command.
```

Comment:
`daytona code`

`open` is an alias of `code`

Reply:
Agreed with @Tpuljak should be `daytona code`